includes/class-perfecty-push-cron-check.php

```diff
@@ -0,0 +1,152 @@
+<?php
+/**
+ * The file that defines the cron check hook
+ *
+ * @link       https://github.com/perfectyorg
+ * @since      1.2.0
+ *
+ * @package    Perfecty_Push
+ * @subpackage Perfecty_Push/includes
+ */
+
+/**
+ * The Cron Check class.
+ *
+ * This class defines all code necessary to monitor that a cron system (wp_cron or system cron) is working.
+ *
+ * @since      1.2.0
+ * @package    Perfecty_Push
+ * @subpackage Perfecty_Push/includes
+ * @author     Giuseppe Foti <foti.giuseppe@gmail.com>
+ */
+class Perfecty_Push_Cron_Check {
```

Comment:
I like this idea of having a separate class :) could we move it to `lib/` instead?